devel/git-town/Portfile

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup            github.com/git-town/git-town 7.6.0 v
+
+description         High-level command-line interface for Git
+long_description    {*}${description}
+
+
+categories          devel
+installs_libs       no
+license             MIT
+maintainers         {karpenko.ca:alexei @akarpenko}
```

Comment:
Please also add `openmaintainer` to your maintainers block, unless there is a specific reason why you should be the only person to manage this port:
```
maintainers         {karpenko.ca:alexei @akarpenko} openmaintainer
```